sources/packages/web/src/views/aest/student/StudentApplicationAppeals.vue

```diff
@@ -0,0 +1,169 @@
+<template>
+  <full-page-container>
+    <template #header>
+      <header-navigator title="Student requests" subTitle="Appeals" />
+    </template>
+    <body-header
+      title="Requested appeals"
+      :recordsCount="applicationAppeals.results?.length"
+      subTitle="Make a determination on requested change(s) that may require a reassessment"
+      class="m-1"
+    >
+      <template #actions>
+        <v-text-field
+          class="v-text-field-search-width"
+          density="compact"
+          label="Search name or application #"
+          variant="outlined"
+          v-model="searchCriteria"
+          @keyup.enter="searchExceptions"
+        >
+          <template v-slot:prependInner>
+            <font-awesome-icon :icon="['fas', 'search']" />
```

Comment:
We are no longer using the `font-awesome-icon` component. Please see below example.
![image](https://user-images.githubusercontent.com/61259237/180271483-7c945871-eeff-43cb-b691-06851d4ad96c.png)